deliverables/README.md

```diff
@@ -21,48 +21,110 @@ For projects on Code Academy, or other external exercises, please just paste a l
 
 ### Section 1: The Front End
 #### Required - Answer these questions in the deliverables file, and complete the following. 
+
 What is the role of HTML in a web page?  
+HTML acts as the skelton of the web page.
+
 What is the role of CSS in a web page?  
-What is the role of Javascript in a web page?  
+CSS acts as the skin of the web page
 
+What is the role of Javascript in a web page?  
+Javascript adds behavior to your webpage allowing the user to interact with it. 
 ---
 
 ### Section 2: HTML and CSS 101
 
 #### meta
 Why do we separate HTML and CSS?  
-
+We seperate the two for effiency, accesbility, device compatibility.
 #### HTML
 What are classes and IDs (and how are they different)?  
-What are elements?  
+ID's are unique, classes are not unique.
+
+What are elements? 
+Elements are everything inbetween the tags.
+
 What are tags?  
+Are keywords surronded by ankle braclets. They come in pairs, a start and end one.
+
 What are attributes?  
+Act as a modifier to the HTML element. They to consist of different tags. 
+
 What are forms?  
+A form allows one to enter data to a web page. They collect the users input. 
+
 What is a div?  
+It groups elements together. The most common method for identyfying structure. 
 
 What are selectors?  
-What are properties?  
+They select specific elements that are unique on the page.
+
+What are properties?
+Properties effect how the code is represeneted.
+
 What are values?  
+Values are specific to the property. For example, red would be a value to the property color.
+
 How do CSS styles for a particular element get inherited? ie. how does an element get its "default" styles?  
-What are two CSS attributes you can change to push an element around on the page?  
-What are the three different ways to include CSS in your project or use CSS to style a particular element?  
+You use ID's to choose the style. If you don't choose the style, the web program should have a default setting unique to the program.
 
+What are two CSS attributes you can change to push an element around on the page? 
+Float and margin can move an element around on the page.
+
+What are the three different ways to include CSS in your project or use CSS to style a particular element?  
+You can use to change font color, change font size and change the type of font.
 ---
 ### Section 3: Git and Github  
 What is Git?  
+A distributed control system. Basically a control system for software development. 
+
 What is SCM?  
+Scheme Implementation.
```

Comment:
That is one meaning of SCM, however, in this case it refers to software configuration management, which is a tool (like git) that allows us to track changes in our code.